config.yaml

```diff
@@ -0,0 +1,11 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+options:
+  django_debug:
+    type: boolean
+    description: "Set the backed by Django API DEBUG setting"
+    default: false
+  django_allowed_hosts:
```

Comment:
Couldn't be a comma-separated list like Paas app charmer does?
https://github.com/canonical/paas-app-charmer/blob/main/examples/django/charm/charmcraft.yaml#L38

Reply:
I changed that to mimic paas app charmer and be a comma-separated list